src/commands/migrate.js

```diff
@@ -17,6 +17,10 @@ exports.builder = (yargs) =>
     .option('from', {
       describe: 'Migration name to start migrations from (excluding)',
       type: 'string',
+    })
+    .option('name', {
+      describe: 'Migration name',
```

Comment:
```suggestion
      describe: 'Migration name. When specified, only this migration will be run',
```

We should document that it's mutually exclusive with `to` & `from`.
We should also throw an error if these mutually exclusive options are provided together.

umzug can accept an array of migration name, we should make sure `name` can be specified more than once:

`sequelize-cli db:migrate --name migration1.js --name migration2.js`

Reply:
c22a291